wire-ios-data-model/Source/Notifications/ObjectObserverTokens/Helpers/StringKeyPath.swift

```diff
@@ -53,7 +53,7 @@ public final class StringKeyPath: Hashable {
     }
 
     public lazy var decompose: (head: StringKeyPath, tail: StringKeyPath?)? = {
-        if 1 <= self.count {
```

Comment:
this one is really messing with my head

Reply:
Yes 😆 I first rewrote it to `self.count > 1`, but it must be `self.count >= 1`, but then I thought it's an `Int` so it can be `self.count > 0`, what is actually meant 😬